pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -266,6 +273,23 @@ public void runSegmentMetrics() {
           nErrors);
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_SEGMENTS_AVAILABLE,
           (nSegments > 0) ? (100 - (nOffline * 100 / nSegments)) : 100);
+
+      // Emit estimated sizes of realtime/offline tables.
+      TableSizeReader.TableSizeDetails tableSizeDetails = _tableSizeReader.getTableSizeDetails(tableName,
```

Comment:
Please emit metrics only if the table exist (we don't want to emit both size for offline / realtime only table)

Reply:
Fixed.